skfda/inference/hotelling/hotelling.py

```diff
@@ -0,0 +1,214 @@
+from skfda.representation import FDataBasis, FData
```

Comment:
This should not be in this PR.

Reply:
The file should be removed from the PR commits if possible, so that the mistake does not appear in history.